tsl/test/sql/CMakeLists.txt

```diff
@@ -110,10 +110,10 @@ endif(CMAKE_BUILD_TYPE MATCHES Debug)
 
 if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
   if(CMAKE_BUILD_TYPE MATCHES Debug)
-    list(APPEND TEST_FILES chunk_utils_internal.sql)
+    list(APPEND TEST_FILES chunk_utils_internal.sql
+         compression_update_delete.sql)
   endif()
-  list(APPEND TEST_FILES compression.sql compression_update_delete.sql
-       compression_permissions.sql)
+  list(APPEND TEST_FILES compression.sql compression_permissions.sql)
```

Comment:
why do we require a debug build for these tests now?

Reply:
Fixed.